Doppler.HtmlEditorApi.Test/DopplerHtmlDocumentTest.cs

```diff
@@ -0,0 +1,49 @@
+using Doppler.HtmlEditorApi.Storage.DapperProvider.Queries;
+using Doppler.HtmlEditorApi.Storage.DapperProvider;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Net;
+using System.Threading.Tasks;
+using Doppler.HtmlEditorApi.Test.Utils;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using System.Text.Json;
+using Xunit.Abstractions;
```

Comment:
What do you think about removing unused dependencies?

```suggestion
using Xunit.Abstractions;
```